Resources/Prototypes/_LostParadise/Catalog/Fills/Boxes/security.yml

```
@@ -0,0 +1,34 @@
+- type: entity
+  name: armband box
+  parent: BoxCardboard
+  id: LPPBoxArmband
+  description: A box of armbands.
+  components:
+  - type: Item
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)